template/Dockerfile.jinja

```diff
@@ -14,10 +14,31 @@ ENV PATH=/venv/bin:$PATH{% if docker %}
 
 # The build stage installs the context into the venv
 FROM developer AS build
-COPY . /context
-WORKDIR /context
+# Requires buildkit 0.17.0
+COPY --chmod=o+wrX . /workspaces/{{ repo_name }}
```

Comment:
Do we want to make this all writable for developer as well as debug? It might not matter that much I guess?